.github/workflows/tests.yml

```diff
@@ -1,8 +1,6 @@
 name: Tests
 on:
   push:
-    branches:
-      - master
```

Comment:
I would say keep this just in case someone pushes to master.

Reply:
This is a restriction, in this case, it means only on master, so with it, branches don't run tests